doc/source/dev/conduct/code_of_conduct.rst

```diff
@@ -0,0 +1,164 @@
+   project-related support, too; not only is a public support request much more
+   likely to result in an answer to a question, it also makes sure that any
+   inadvertent mistakes made by people answering will be more easily detected
+   and corrected.
+
+2. Be empathetic, welcoming, friendly, and patient. We work together to resolve
+   conflict, assume good intentions, and do our best to act in an empathetic
+   fashion. We may all experience some frustration from time to time, but we do
+   not allow frustration to turn into a personal attack. A community where
+   people feel uncomfortable or threatened is not a productive one. We should
+   be respectful when dealing with other community members as well as with
+   people outside our community.
+
+3. Be collaborative. Our work will be used by other people, and in turn we will
+   depend on the work of others. When we make something for the benefit of the
+   project, we are willing to explain to others how it works, so that they can
+   build on the work to make it even better. Any decision we make will affect
+   users and colleagues, and we take those consequences seriously when making
+   decisions.
+
+3. Be inquisitive. Nobody knows everything! Asking questions early avoids many
+   problems later, so questions are encouraged, though they may be directed to
+   the appropriate forum. Those who are asked should be responsive and helpful,
+   within the context of our shared goal of improving SciPy project code,
+   documentation and other content.
+
+4. Be careful in the words that we choose. Whether we are participating as
+   professionals or volunteers, we value professionalism in all interactions,
+   and take responsibility for our own speech. Be kind to others. Do not insult
+   or put down other participants. Harassment and other exclusionary behaviour
+   are not acceptable. This includes, but is not limited to:
+
+    - Violent threats or language directed against another person.
+    - Sexist, racist, or otherwise discriminatory jokes and language.
+    - Posting sexually explicit or violent material.
+    - Posting (or threatening to post) other people's personally identifying information ("doxing").
+    - Sharing private content, such as emails sent privately or non-publicly, or unlogged forums such as IRC channel history.
+    - Personal insults, especially those using racist or sexist terms.
+    - Unwelcome sexual attention.
+    - Excessive or unnecessary profanity.
+    - Repeated harassment of others. In general, if someone asks you to stop, then stop.
+    - Advocating for, or encouraging, any of the above behaviour.
+
+
+Diversity Statement
+-------------------
+
+The SciPy project welcomes and encourages participation by everyone. We are
+committed to being a community that everyone feels good about joining. Although
+we may not be able to satisfy everyone, we will always work to treat everyone
```

Comment:
we may not always be able to accommodate each individual's preferences, we try our best to treat everyone kindly.